perf_test/graph/KokkosGraph_color.cpp

```diff
@@ -181,6 +184,8 @@ void run_experiment(
     kh.set_verbose(true);
   }
 
+  std::cout << "algorithm: " << algorithm << std::endl;
```

Comment:
Assume this is temporary.

Reply:
Never mind previous comment, this is in testing

Reply:
I added guards and now you need to request these outputs explicitly, it might actually make sense to have some special debugging flag to get that level of output details?